src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -137,6 +154,14 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 			" in the default locations",
 	)
 	mgr.BindFlag("config_file", command.Flags().Lookup("config"))
+	command.Flags().StringSliceVarP(
+		&seeds,
+		"seeds",
+		"s",
+		[]string{},
+		"A list of seed nodes to connect to, in the format"+
+			" <host>:<port>+<id>",
```

Comment:
Is it critical that they know the node id or is this optional?

Reply:
Redpanda seems to use the `node_id`, would be nice if it didn't. And perhaps safer.

Reply:
#190